examples/slsa-demo/README.md

```diff
@@ -0,0 +1,32 @@
+# SLSA L2 Demo with Tekton
+This repo shows how Tekton can be used to acheive SLSA L2 requirements.
+
+### To run the demo
+* Clone the repo.
+* Edit env.sh to add appropriate values for running the demo.
+* Run setup.sh which sets up the environment.
+* Run run_pipeline.sh.
+* After the pipeline run is done run provenance_extractor.sh
+
+### How the demo works
+#### setup.sh
+
+setup.sh based on the settings specified in env.sh, sets up
+* Creates a Kubernetes cluster on Google Cloud.
+* Sets up Artifact registry on Google Cloud.
+* Sets up the KMS on Google Cloud.
```

Comment:
I'm happy to move over to `docs/vendor/gcp` -- but leaving it here for the moment so that you can easily diff the changes I made to address your first round of feedback.

Reply:
moving to `docs/vendor/...`